### ISSUE TITLE
Accessing Module in site-packages

### ISSUE THREAD

**Issue by neil-benn** [ninia/jep#349]
Hello,

  I'm working on a windows machine with Python 3.8 using jep integrated into an application.  If I'm calling pure Python code it works just fine - however I'm trying to call an external modules and this is the issue.  Firstly I've installed the module as administrator using the following java code:

```
try (Interpreter interp = new SharedInterpreter()) {
	populateGlobalNamespace(interp, model);
	interp.exec(model.getExportSettings().getScriptResultFormat().getScript());
} catch (PythonScriptException | JepException e) {
	throw new ExportException("Failed to execute the script " + e.getMessage(), e);
}
```

  It doesn't do anythign special and the populate globsal namespace just puts a bunch of string variabkles in teh global namespace for later use by the script write.

  On the python side I first install the third party module as administrator on a windows machine so it is globally available - though I also tried it as user too.

pip install pyexcel

  I've then got some test code:

```
from openpyxl import Workbook

workbook = Workbook()
sheet = workbook.active

sheet["A1"] = "hello"
sheet["B1"] = "world!"

workbook.save(filename="hello_world.xlsx")
```

  This executes with python testexcel.py just fine,  However when I try the same thing and execute it with the same python runtime in jep I get:

<class 'ModuleNotFoundError'>: No module named 'openpyxl'

  Is there something I need to do to get jep to correctly load third party modules via site-packages?  Note other site-packages work such as LZ4, CV2 and so on - is the native component of my python excel site-package that causes the problem?

  Thanks.

Cheers,

Neil

**Comment by ndjensen** [comment]
No, there's nothing special you have to do to get packages from site-packages imported.  Lots of times when we see this there's more than one Python installed on the system and Jep is using the Python that doesn't have the site-package installed.  Can you check if that's happening?  I think Jep will use the Python specified in PATH, but it might use the one specified by PYTHONHOME if present.

**Comment by neil-benn** [comment]
Hello,

  Thanks for the response - I've checked that and double checked it.  In favt we specifically set the python path in our code to avoid the python path or path causing problems:

```
  System.load(Path.of(pythonSettings.getPythonPath(), "python38.dll").toAbsolutePath().toString());
  MainInterpreter.setJepLibraryPath(Path.of(
    pythonSettings.getPythonPath(), 
   "Lib", 
   "site-packages", 
   "jep", "jep.dll")
   .toAbsolutePath()
   .toString());
```

  I've also done a test by using pip3 install pandas (as admin) and then trying to import pandas both in my python3 install (c:\program files\python38) from the command line and from a script running through jep and it works just fine.  I do have:

PYTHONHOME set to c:\program files\python38
The above Jep setup code set to c:\program files\python38
PATH also has c:\program files\python38 setup

  There are no other python installs on the computer that I have installed.  I also checked the version I get from runnign jep:

`3.8.1 (tags/v3.8.1:1b293b6, Dec 18 2019, 23:11:46) [MSC v.1916 64 bit (AMD64)]
`
  With the version I get from the command prompt:

`'3.8.1 (tags/v3.8.1:1b293b6, Dec 18 2019, 23:11:46) [MSC v.1916 64 bit (AMD64)]'`

  I did this using python's sys.version call.

  I can't find any evidence that I'm using different python installs - are there any other ideas?  I could write a very simple test case if it will help as the jep I'm using is embedded in a larger application.
